packages/login_client/lib/src/login_client.dart

```diff
@@ -206,11 +213,12 @@ class LoginClient extends http.BaseClient {
   /// Disposes the [LoginClient].
   Future<void> dispose() => _onCredentialsChanged.close();
 
-  // ignore: use_setters_to_change_properties
   @visibleForTesting
+  // ignore: use_setters_to_change_properties, public_member_api_docs
```

Comment:
`use_setters_to_change_properties` does not have to be ignored here anymore, they probably fixed some false positives.